spec/controllers/queue_items_controller_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'spec_helper'
+
+describe QueueItemsController do
+  describe 'GET index' do
+    it 'sets @queue_items with authentication' do
+      user = Fabricate(:user)
+      session[:user_id] = user.id
+      @queue_items = user.queue_items
```

Comment:
You should not use instance variables in specs. This will work fine as a local variable.